dll/shellext/cabview/precomp.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * PROJECT:     ReactOS CabView Shell Extension
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
```

Comment:
Please use the "or-later" version (see our code base).

Reply:
I copied this from somewhere but I don't remember where, this was some time last year probably.